site/content/contribute/getting-started/contributions-without-ticket.md

```diff
@@ -0,0 +1,23 @@
+---
+draft: true
+title: "Contributions Without Ticket"
+date: 2017-08-20T12:33:36-04:00
+weight: 1
+subsection: Getting Started
+---
+
+# Contributions Without Ticket
+
+Contributions for minor corrections and improvements without a ticket are welcome.
+
+It is acceptable to submit a PR for a bug or incremental improvement, with less than 20 lines of code change, without a Help Wanted ticket if the change to existing behaviour is minor. All pull requests submitted without a corresponding ticket will first be reviewed by a core team product manager. Some examples of minor corrections and improvements include:
+
+ - [Fix a formatting error in help text](https://github.com/mattermost/mattermost-server/pull/5640)
+ - [Fix success typo in Makefile](https://github.com/mattermost/mattermost-server/pull/5809)
+ - [Fix broken Cancel button in Edit Webhooks screen](https://github.com/mattermost/mattermost-server/pull/5612)
+ - [Fix Android app crashing when saving user notification settings](https://github.com/mattermost/mattermost-mobile/pull/364)
+ - [Fix recent mentions search not working](https://github.com/mattermost/mattermost-server/pull/5878)
+ 
+Larger pull requests, including those that require a change to documentation or user expectations, [require a Help Wanted issue opened by the core team](http://docs.mattermost.com/process/help-wanted.html) so that the change can be tested, documented and supported.
```

Comment:
>tested, documented and supported

That sounds like the BS reason. We can do all that after a PR is submitted. The real reason to me is we want the product to have a unified vision, and to do that large changes need to discussed and agreed upon by the Mattermost core team.

The text that was there before is much more genuine IMO:

>Core committers who review the PR are entitled to reject it if they feel it significantly changes behavior or user expectations, which requires a Help Wanted ticket opened by the core team. This helps ensure that everything going into the project aligns with a unified vision.

Reply:
I just copy-pasted what we had. We can update that. Wasn't aware (or had forgotten) about the updated text.

Reply:
Ah ok, I thought it was new text you had written 👍